task_chain/autokitteh.yaml

```diff
@@ -0,0 +1,25 @@
+# This YAML file is a declarative manifest that describes the setup
+# of an AutoKitteh project that ...TODO...
```

Comment:
TODO

Reply:
Ack, already noted in the PR description. I'll do this (and a README) when the code structure is acceptable.